frontend/src/utils/Utils.ts

```diff
@@ -59,3 +59,9 @@ export const generateSortFn =
 export const isObjEmpty = (o: object): boolean => {
   return Object.keys(o).length === 0;
 };
+
+// Retrieved from: https://stackoverflow.com/a/11818658
+export const toFixed = (num: number, fixed: number): string => {
```

Comment:
nit: i think `twoSigFigs `or `truncate ` make more sense, what do you think? toFixed doesnt mean much to me
Ill leave it up to you!